data/locale/en-US.ini

```diff
@@ -11,6 +11,11 @@ RestartCEF="Restart CEF"
 BrowserSource="Browser"
 CustomFrameRate="Use custom frame rate"
 RerouteAudio="Control audio via OBS"
+WebpageControlLevel="Webpage permissions"
+WebpageControlLevel.Level.None="Don't allow any access"
+WebpageControlLevel.Level.ReadOnly="Allow read-only access"
+WebpageControlLevel.Level.Basic="Allow basic control (Start/Stop replay buffer, etc.)"
```

Comment:
Oh, you're right - I meant "Save replay buffer" when it says "Start/Stop", will fix

I'm personally not sure it would be a good idea to give the user even more possibilities, though I have no strong feelings about that

Reply:
Yep, I have no strong feelings either way myself.

Reply:
Fixed the wording on the current behaviour, left the levels how they are for now.

Maybe someone else has opinions on the above ^

Reply:
Update:
Thought about this, an "Advanced" level probably is a good idea.
That could be used as well for say switching scenes (which is something I plan on adding once the initial functionality is merged), and only the most dangerous controls (aka start/stop of the outputs) would be under "all".

The levels would then be:
- None
- Read-Only
- Basic (functions that shouldn't be able to break stuff)
- Advanced (functions that are able to break the current setup in non-fatal ways)
- All (functions that are able to break everything)
 
I still think 5 options might be a little much but probably a drawback worth taking.

If you agree, I'd add this.

Reply:
I would say that's a worthwhile distinction.